aicsimageio/tests/readers/test_lif_reader.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from io import BytesIO
+
+import numpy as np
+import pytest
+from dask.diagnostics import Profiler
+from psutil import Process
+
+from aicsimageio import exceptions
+from aicsimageio.readers.lif_reader import LifReader
+
+
+@pytest.mark.parametrize(
+    "filename, "
+    "expected_shape, "
+    "expected_dims, "
+    "expected_dtype, "
+    "select_scene, "
+    "chunk_dims, "
+    "expected_chunksize, "
+    "expected_task_count",
+    [
+        # Expected task counts should be each non chunk dimension size multiplied againest each other * 2
+        (
+            "s1_t1_c2_z1.lif",
```

Comment:
Can you change these file names to match the rest of the files, `s_1_t_1_c_2_z_1.lif`